spec/world_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative '../lib/world'
+require_relative '../lib/cell'
+
+RSpec.describe World do
+
+  let (:world) {World.new}
+
+  it "returns no live cells if there is only 1 cell" do
+    cell = Cell.new([1,1])
+    world = World.new([cell])
+    world.evolve
+    expect(world.cells).to eq([])
+  end
+
+  it "returns cell with 2 neighbours" do
+    first_cell = Cell.new([1,1])
+    second_cell = Cell.new([2,2])
+    third_cell = Cell.new([0,0])
+    world = World.new([first_cell, second_cell, third_cell])
+    world.evolve
+    expect(world.cells).to eq([first_cell])
```

Comment:
I feel the assertion here doesn't match up that clearly with the name of the test. 

Can the name be changed to align better with the assertion or vice-versa? For example, the name of the test mentions neighbours and 2, whereas the assertion has no notion of this. Could you use your `world.count_live_neighbours` method to clarify this a little?